client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -42,6 +42,19 @@
             ng-show="_editable"
             translate>Publish</button>
 
+        <div class="navbtn dropdown pull-left" dropdown>
+            <button class="dropdown-toggle" dropdown-toggle><i class="icon-dots-vertical"></i></button>
+            <div class="dropdown-menu">
+                <ul>
+                    <li ng-controller="TemplateActionsController as template">
```

Comment:
The name "template" for a controller could be a bit misleading, perhaps think of another name...but it's a minor thing.